models/spring-ai-openai/src/test/java/org/springframework/ai/openai/api/OpenAiApiIT.java

```diff
@@ -76,11 +75,10 @@ void chatCompletionStream() {
 	}
 
 	@Test
-	@Disabled("The reasoning_effort option is only available in o1 models.")
 	void validateReasoningTokens() {
 		ChatCompletionMessage userMessage = new ChatCompletionMessage(
```

Comment:
This test result isn't consistent with the current settings as the reasoning tokens end up being "0" in some cases when the model doesn't seem to use reasoning tokens for simple problems unless being more specific with the prompt message.